dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/view/SiteView.java

```diff
@@ -0,0 +1,40 @@
+package com.dotcms.rest.api.v1.secret.view;
+
+import com.dotcms.security.secret.Secret;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import java.util.Map;
+
+public class SiteView {
+
+    private final String id;
+    private final String name;
+
+    @JsonInclude(Include.NON_NULL)
+    private final Map<String, Secret> secrets;
+
+    public SiteView(final String id, final String name) {
+        this.id = id;
+        this.name = name;
+        this.secrets = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)